### PR TITLE
Q4 Roadmap

### PR DESCRIPTION
**Pull request by breathingdust**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

#### Description

This PR outlines the maintainer teams focus for the coming quarter. It also adds ElastiCache as a Core Service.

### CODE REVIEW

File: ROADMAP.md [hunk 1]
```diff
@@ -1,112 +1,101 @@
-# Roadmap: August - October 2020
+# Roadmap:  November 2020 - January 2021
 
 Every few months, the team will highlight areas of focus for our work and upcoming research.
 
-We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. When community pull requests exist for a given item, we will prioritize working with the original authors to include their contributions. If the author can no longer take on the implementation, HashiCorp will complete any additional work needed.
+We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist, or the original contributors are not available we will create the resources and implementation ourselves.
```
Comment (maryelizbeth) [inline ROADMAP.md:5]:
```suggestion
We select items for inclusion in the roadmap from the Top 10 Community Issues, [core services](docs/CORE_SERVICES.md), and internal priorities. Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist or the original contributors, are not available we will create the resources and implementation ourselves.
```

Comment (maryelizbeth) [reply ROADMAP.md:5]: Super small change here! feel free to disregard

File: ROADMAP.md [hunk 2]
```diff
@@ -1,112 +1,101 @@
...
 
 Each weekly release will include necessary tasks that lead to the completion of the stated goals as well as community pull requests, enhancements, and features that are not highlighted in the roadmap. To view all the items we've prioritized for this quarter, please see the [Roadmap milestone](https://github.com/terraform-providers/terraform-provider-aws/milestone/138).
 
-Where community sourced contributions exist we will work with the authors to review and merge their work. Where this does not exist we will create the resources and implementation ourselves.
+This roadmap does not describe all the work that will be included within this timeframe, we will include other work as events occur but it does describe our focus.
```
Comment (maryelizbeth) [inline ROADMAP.md:9]:
```suggestion
This roadmap does not describe all the work that will be included within this timeframe, but it does describe our focus. We will include other work as events occur .
```

Comment (maryelizbeth) [reply ROADMAP.md:9]: revised sentence order for clarity

**Review by maryelizbeth — CHANGES_REQUESTED** [review]
Some tiny suggestions but otherwise LGTM

File: ROADMAP.md (hunk 1 above)
Comment (ewbankkit) [inline ROADMAP.md:5]: Change `core services` to `Core Services` to match capitalization later in the document.

**Pull request merged**